locksmith/src/payment/paymentProcessor.ts

```diff
@@ -358,6 +365,11 @@ export class PaymentProcessor {
               transactionHash,
               chain: network,
             })
+
+            await this.stripe.paymentIntents.capture(paymentIntentId, {
```

Comment:
Capture only when/if blockchain transaction was submitted. This will prevent charges for which the blockchain tx was not sent.

There is a small risk that the bockchain tx fails _after_ it has been sent (mostly `MAX_KEY`) if there is a concurrent  transaction